tests/testthat/test-GLV.R

```diff
@@ -9,4 +9,10 @@ test_that("simulateGLV", {
     expect_true(is.matrix(InterMatx))
     expect_equal(dim(InterMatx), c(4,1000))
     expect_s4_class(SEobject, "SummarizedExperiment")
+
+    #check norm = TRUE
+    SEobject2 <- miaSim:::simulateGLV(N = 4, A = powerlawA(n = 4, alpha = 2), tend = 1000, norm = TRUE)
+    InterMatx2 <- assay(SEobject)
+    expect_true(is.matrix(InterMatx2))
```

Comment:
There is also a test to expect type or class (expect_type?). That should be better than is.true(is.matrix())